libs/wingc/src/type_check.rs

```diff
@@ -636,6 +636,28 @@ impl TypeRef {
 			_ => false,
 		}
 	}
+
+	pub fn is_serializable(&self) -> bool {
+		match **self {
+			Type::Resource(_) => false,
+			Type::Enum(_) => true,
```

Comment:
For now lets leave Enum as false
```suggestion
			Type::Enum(_) => false,
```